frontend/src/components/HeatMap.tsx

```
@@ -0,0 +1,57 @@
+  const { userKey } = useParams()
+  const [data, setData] = useState<HeatmapData | null>(null)
+  const canvasRef = useRef<HTMLCanvasElement | null>(null)
+  const [imageLink, setImageLink] = useState('')
+  const [username, setUsername] = useState('')
+  const [privateContributor, setPrivateContributor] = useState(false)
+  const theme = 'blue'
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const result = await fetchHeatmapData(userKey)
+      if (result && result.contributions) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)